lib/charliecloud.py

```diff
@@ -33,7 +36,11 @@
    # Lark is additionally messy because there are two packages on PyPI that
    # provide a "lark" module.
    import lark   # ImportError if no such module
-   lark.Visitor  # AttributeError if wrong module
+   lark.Visitor # AttributeError if wrong module
+   lark_path = os.path.abspath(lark.__file__)
+   desired_path = os.path.dirname(os.path.abspath(__file__)) + "/lark_parser.egg/lark/__init__.py"
+   if lark_path != desired_path:
+       sys.exit("Lark found was not embedded")
```

Comment:
```suggestion
       sys.exit("lark-parser found was not embedded")
```

Reply:
We should maybe refer to it as 'lark-parser' to be consistent with other warnings and errors.